dali/test/python/test_operator_get_property.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nvidia.dali import pipeline_def
+import os
+from nvidia.dali import fn
+from test_utils import get_dali_extra_path
+
+test_data_root = get_dali_extra_path()
+images_dir = os.path.join(test_data_root, 'db', 'single', 'png')
+
+@pipeline_def(batch_size=2,num_threads=1,device_id=0)
+def pipe():
+    read, _ = fn.readers.file(file_root=images_dir)
+    return fn.get_property(read, key="source_info")
+
+def main():
+    p=pipe()
+    p.build()
+    out=p.run()
+    nda=out[0].as_array()
+    print(nda.tostring())
```

Comment:
I'm not sure if this should be the way. Maybe rather:
```
for t in out[0]:
    print(np.array(t).tostring())
``?

Reply:
That's a draft test ;) I just wanted to check if the CI passes. This whole file is going to change

Reply:
I'm not commenting the test but the API and the recommended way to use it.
If we want the tensors returned by this OP uniform and the whole batch easily convertible to tensor? Or we think that the user will obtain the values per sample so `as_array()` won't be called.

Reply:
Done, I went with your suggestion